pkg/nsx/services/store.go

```diff
@@ -166,3 +190,33 @@ func queryRule(service *SecurityPolicyService, wg *sync.WaitGroup, fatalErrors c
 		}
 	}
 }
+
+func decrementPageSize(pageSize *int64) {
+	*pageSize -= 100
+	if int(*pageSize) <= 0 {
+		*pageSize = 1
```

Comment:
it seems 1 is too small for the page size.

Reply:
I think it seldom hit here, unless error appears when pagesize=100, it can't happen.